pontoon/base/views.py

```diff
@@ -377,6 +377,53 @@ def get_translations_from_other_locales(request):
     return JsonResponse(payload, safe=False)
 
 
+@utils.require_AJAX
+# need to define preferred_source_locale
+def get_sibling_entities(request):
+    """Get entities preceding and succeeding the current entity"""
+    print("TEST-------------------")
+    try:
+        entity = int(request.GET["entity"])
+        locale = request.GET["locale"]
+    except (MultiValueDictKeyError, ValueError) as e:
+        return JsonResponse(
+            {"status": False, "message": "Bad Request: {error}".format(error=e)},
+            status=400,
+        )
+
+    entities = Entity.objects.get(id=entity)
+
+    preceding_entities = (
+        Entity.objects.filter(resource=entities.resource, obsolete=False)
+        .order_by("order")
+        .filter(order__gt=entities.order)
```

Comment:
Preceding entities have an `order`that is **less than (`lt`)** the order of the current entity, so `order__lt=entity.order`. And inversely below for the `succeeding_entities`.

Reply:
I was tired :)